backend/src/yeomanui.ts

```diff
@@ -82,6 +84,9 @@ export class YeomanUI {
     this.currentQuestions = {};
     this.setGenFilter(genFilter);
     this.customQuestionEventHandlers = new Map();
+    if (vscode.workspace && vscode.workspace.workspaceFolders && vscode.workspace.workspaceFolders.length > 0) {
+      YeomanUI.cwd = vscode.workspace.workspaceFolders[0].uri.path;
+    }
```

Comment:
it is better to pass first workspace folder object or even it's uri.path to the yeomanui constructor.
We do not need dependency to vscode here.

When you do it please use lodash

Reply:
1. pass path to the constructor.
2. why only workspaceFolders[0] ? is this by design (if it's multi-root worksapce the take the 'root ws' folder)?

Reply:
workspaceFolders[0] holds the current active workspace, do u see a use case that we need another workspace?

Reply:
sounds good to me.